_layouts/english.html

```diff
@@ -0,0 +1,13 @@
+---
+layout: default
+---
+
+<div class="cover-photo">
+  <img src="/assets/img/2019_all.png" alt="Mitou Junior Photo in 2019" class="top-img" style="margin-bottom: -8px;">
```

Comment:
English speakers would directly access `/english` page, not via top page, so I add the cover-photo section at the top of English page. 📸 ✨